cf/read_write/netcdf/netcdfread.py

```diff
@@ -298,9 +313,7 @@ def _create_Data(
             ncdimensions: sequence of `str`, optional
                 The netCDF dimensions spanned by the array.
 
-                .. versionadded:: 3.14.0
-
-            units: `str`, optional
+                .. versionadded:: 3.14.fill_value:
```

Comment:
```suggestion
                .. versionadded:: TODOCFAVER
```
otherwise I am not sure what the `fill_value` entry is meant to represent / be a placeholder for?